pennylane_sf/gbs.py

```diff
@@ -82,6 +85,9 @@ def __init__(self, wires, *, shots=None, cutoff_dim, use_cache=False, samples=No
         self._WAW = None
         self.Z_inv = None
 
+        warnings.warn("strawberryfields.gbs will be deprecated for pennylane>=v0.23.0."
+                      "If you would like to use pennylane must be <=v022.0", DeprecationWarning)
```

Comment:
Maybe we can also encourage users to reach out if they want to keep using this device.
```suggestion
                      "If you would like to use this device now and in the future, downgrade PennyLane to <=v022.0 and create a related issue on the GitHub page of PennyLane-SF.", DeprecationWarning)
```